code-challenges.test.js

```diff
@@ -10,37 +12,104 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+
+
+//   ***Hate to end on such a low note, but my family has decided to pay me a surprise visit and now I barely have time to think. Apologies for the subpar work on this one.***
+
+
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
 
+// input: an array of objects
+// output: an array with a sentence about each person with their name capitalized.
+// Pseudo: Map across the array, using a regular expression powered .replace method to capitalize the names. after that, need to stick the output in a concatenated string and return those as an array.
+
+
+
+
+
+describe('heartOfGold', () => {
+  it('return an array with a sentence about each person with their name capitalized', () => {
+    expect(heartOfGold(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  });
+});
+
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
+
+const heartOfGold = (towel) => {
+  const properPeople = towel.map(person => {
```

Comment:
Nice job! This is easy to follow and a very step-by-step approach. Definitely encourage you to refactor if it makes sense. 👏🏽